extensions/binding/org.eclipse.smarthome.binding.tradfri/OSGI-INF/TradfriDiscoveryParticipant.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.tradfri.internal.discovery.TradfriDiscoveryParticipant">
```

Comment:
Name of this component is ` name="org.eclipse.smarthome.binding.tradfri.internal.discovery.TradfriDiscoveryParticipant"` while the name in the handler is `binding.tradfri`